Example/Pods/Target Support Files/Pods-URBNSwiftAlert_Example/Pods-URBNSwiftAlert_Example-acknowledgements.markdown

```diff
@@ -3,7 +3,7 @@ This application makes use of the following third party libraries:
 
 ## URBNSwiftAlert
 
-Copyright (c) 2017 Kevin Taniguchi <ktaniguchi@urbn.com>
```

Comment:
@KVTaniguchi Change this to `Urban Outfitters, Inc.`, `URBN Inc.` isn’t a legal entity

Reply:
updated